src/avl.c

```diff
@@ -486,26 +493,31 @@ int print_avl(avl_p root, avl_p parent)
  * ------
  * root: pointer to the root node pointer of an AVL tree
  * val : value to search
- * stop: stop if val is found
+ * stop: (TRUE) stop if Key is found and return val. (FALSE) continue visiting of tree and return last key found
```

Comment:
Can we have only 1 param `stop` as this:

TRUE: stop if Key is found and return val.
FALSE: continue visiting of tree, print debug info and return 1.

Reply:
sorry jarun, but not undestand what you want... sorry

Reply:
I meant remove the parma `show` and if `stop` is false, only then show the info.

Reply:
correct now ;-)

but for stop=False, the function return the last val found (it's more easy to implement)